pkg/controller/controller_broker_test.go

```diff
@@ -395,7 +395,7 @@ func TestReconcileClusterServiceBrokerExistingClusterServiceClassDifferentBroker
 	events := getRecordedEvents(testController)
 	assertNumEvents(t, events, 1)
 
-	expectedEvent := corev1.EventTypeWarning + " " + errorSyncingCatalogReason + ` ClusterServiceBroker "test-broker": Error reconciling ClusterServicePlan (K8S: "PGUID" ExternalName: "test-plan"): ClusterServiceBroker "test-broker": ClusterServicePlan "test-plan" already exists for Broker "notTheSame"`
+	expectedEvent := corev1.EventTypeWarning + " " + errorSyncingCatalogReason + ` Error reconciling ClusterServicePlan (K8S: "PGUID" ExternalName: "test-plan"): ClusterServicePlan (K8S: "PGUID" ExternalName: "test-plan") already exists for Broker "notTheSame"`
```

Comment:
can you change this to a `fmt.Sprintf` for more clarity?

Reply:
Sure can on a follow-up cl!